.circleci/config.yml

```diff
@@ -47,10 +47,11 @@ jobs:
           command: kubectl create -f /cncf/rbac/
           environment:
             KUBECONFIG: /cncf/data/kubeconfig
-      - persist_to_workspace:
-          root: /cncf/data
+      - run: find /cncf/data
```

Comment:
Are these `find /cncf/data` simply for debug purposes?

Reply:
You are correct, let me remove them.